docs/src/main/tut/auth.md

```diff
@@ -55,6 +55,13 @@ val middleware: AuthMiddleware[IO, User] =
   AuthMiddleware(authUser)
 ```
 
+Note: In the above, the default apply method of `AuthMiddleware` will consume all requests either unmatched, or
+not authenticated by returning an empty response with status code 401 (Unauthorized). This mitigates
+a kind of reconnaissance called "spidering", useful for white and black hat hackers to enumerate
+your api for possible unprotected points. To allow fallthrough as normal,
```

Comment:
Most people think falling through is "normal."  I'd rephrase this.

Reply:
Maybe just as "to allow fallthrough"?

Reply:
:+1: